File: app/src/main/java/com/example/filip/movielist/ui/favorite/view/FavoriteMoviesView.java
package com.example.filip.movielist.ui.favorite.view;

import com.example.filip.movielist.pojo.MovieListModel;

import java.util.List;

/**
 * Created by Filip on 03/05/2016.
 */
public interface FavoriteMoviesView {
    void setAdapterItems(List<MovieListModel> moviesToLoad);

    void onFailedToLoadFavoriteMoviesFromDatabase();
}

File: app/src/main/java/com/example/filip/movielist/view/FavoriteMoviesView.kt
package com.example.filip.movielist.view

import com.example.filip.movielist.model.MovieListModel

/**
 * Created by Filip Babic @cobe
 */
interface FavoriteMoviesView : BaseView {

    fun setAdapterItems(items: MutableList<MovieListModel>?)

    fun showNoMoviesError()
}